### ISSUE TITLE
bug: TypescriptAddMissingImports E464: Ambiguous use of user-defined command

### ISSUE THREAD

**Issue by rofrol** [LazyVim/LazyVim#813]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.9.0

### Operating system/version

MacOS 13.4

### Describe the bug

When I have some missing imports in typescript file, I try to auto add them with `:TypescriptAddMissingImports`

https://github.com/LandonSchropp/dotfiles/blob/f4eaaa1ef84f197753410582ace871b87806e712/config/nvim/lua/plugins/lspconfig.lua#LL31C25-L31C25

### Steps To Reproduce

1. Create some typescript file with missing imports
2. Run `:TypescriptAddMissingImports`

### Expected Behavior

Imports are added.

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by folke** [comment]
That's really odd. Is tsserver even connected in that buffer?

Do know that that command is only available in typescript buffers, not in other buffers

**Comment by rofrol** [comment]
> That's really odd. Is tsserver even connected in that buffer?
How to check it?
> Do know that that command is only available in typescript buffers, not in other buffers
Yes, `TypescriptAddMissingImports` is not autocompleted in non-ts buffers.
> You can also call organize imports with code actions btw
how?

**Comment by folke** [comment]
Wait, so you do get that command in auto-completion? But get an error when executing it? I thought the command was not available to you for some reason.

Code actions: `<leader>ca`

**Comment by rofrol** [comment]
>so you do get that command in auto-completion? But get an error when executing it?

Yes

**Comment by folke** [comment]
That error happens when there's another command that is similar to that one.
On the cmdline, when you press `<tab>` do you get any completion for another command?

**Comment by rofrol** [comment]
Yes

![image](https://github.com/LazyVim/LazyVim/assets/28787/dd95cc6c-2bce-4419-b21f-cf01037bc034)

**Comment by rofrol** [comment]
I just did fresh install with LazVim-starter. Enabled only `lazyvim.plugins.extras.lang.typescript` and got the same error.

**Comment by rofrol** [comment]
Works when run with `<Leader>ca` and selected `Add missing imports`

**Comment by folke** [comment]
The error simply means that you just execute `Typescript` and there is more than one command that starts with Typescript. This is all just basic Vim tbh.

**Comment by rofrol** [comment]
You are right. I moved command line widget to bottom as it is in vim because I didn't see what was replaced when doing `%s/one/two/gc`, but then the completions where displayed on the center and I thought it was already chosen.
